generators/app/src/write.js

```diff
@@ -1,15 +1,21 @@
 var _ = require('lodash');
 var path = require('path');
+var merge = require('merge');
 var changeCase = require('change-case');
 
 module.exports = function (Generator) {
   Generator.prototype.writing = {
     app: function () {
-      this.installTemplateFolder({
-        generator: this,
-        destination: path.join(this.config.get('clientSideFolder'), '..'),
-        fileMacros: {'_': changeCase.paramCase(this.name)}
-      });
+    	var globOptions;
+		if (!this.config.get('swagger')) {
+			globOptions = {ignore: '**/api/**'};
```

Comment:
This is a clever addition. ;) Ideally you would have refactored the selector to be an object instead since argument spam isn't good... But don't worry about this one for now